### DISCUSSION TITLE
supabase CORS errors

### DISCUSSION THREAD

**Discussion by EdgarsLv** [supabase/supabase#8821] [General]
What is going on? 

![image](https://user-images.githubusercontent.com/56837461/188738853-8a0affe4-4878-42fb-9765-cb436c05c849.png)

I'm constantly getting kicked out from dashboard and have to re-login. Can't navigate in dashboard menu and getting permission errors. Even one of my projects got lost and had to re-login to get it back...

It's only me or someone else have issues?

**Comment by EdgarsLv** [comment]
Impossible to work.
![image](https://user-images.githubusercontent.com/56837461/188740325-2ef7ba3b-cca3-460a-98c4-59a27638e57d.png)

**Comment by GaryAustin1** [comment]
Hopefully you have contacted support of have seen this:
https://status.supabase.com/
![image](https://user-images.githubusercontent.com/54564956/188760903-6e9ccc90-3400-43f2-b992-55d45188142e.png)
